abfe_tutorial/python_tutorial.ipynb

```diff
@@ -0,0 +1,1186 @@
+{
```

Comment:
<blockquote>First we must load the chemical models between which we wish to calculate free energies.</blockquote><p>First we must load the chemical models for which we wish to calculate the binding free energy.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>